docs/jwst/skymatch/README.rst

```diff
@@ -137,6 +137,13 @@ The ``skymatch`` step has the following optional arguments:
   final "good" bits. For example, both ``4,8`` and ``4+8``
   are equivalent to setting ``dqbits`` to 12.
 
+  Finally, instead of integers, the JWST mnemonics for the bitflags may
+  be used. For example, all the following specifications are equivalent:
+
+  `12 == 4 + 8 == "4, 8" == "JUMP_DET, DROPOUT"`
```

Comment:
Cool - so the user can specify it either as numbers or mnemonics? Fancy!

Reply:
So it looks like the only doc updates included here are for `skymatch`. What about `outlier_detection` and/or `resample`?